inc/wp-components/classes/class-byline-wrapper.php

```diff
@@ -42,15 +42,17 @@ public function default_config(): array {
 	 * @return self
 	 */
 	public function post_has_set(): self {
-
 		$byline_components = [];
 
-		// Use guest author.
+		// Use guest authors, if Coauthors is enabled, or use
```

Comment:
probably better without the first comma: 

`// Use guest authors if Coauthors is enabled,`